Realm.xcodeproj/project.pbxproj

```diff
@@ -246,6 +236,18 @@
 		3FEB38401E70AC8800F22712 /* ObjectCreationTests.mm in Sources */ = {isa = PBXBuildFile; fileRef = 3FEB383C1E70AC6900F22712 /* ObjectCreationTests.mm */; };
 		3FEC4A3F1BBB18D400F009C3 /* SwiftSchemaTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3FEC4A3D1BBB188B00F009C3 /* SwiftSchemaTests.swift */; };
 		3FF3FFAF1F0D6D6400B84599 /* KVOTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5D660FFF1BE98D880021E04F /* KVOTests.swift */; };
+		49B299882409338200B7319C /* RLMAppCredentials.mm in Sources */ = {isa = PBXBuildFile; fileRef = 49B299842409338200B7319C /* RLMAppCredentials.mm */; };
+		49B299892409338200B7319C /* RLMAppCredentials.h in Headers */ = {isa = PBXBuildFile; fileRef = 49B299852409338200B7319C /* RLMAppCredentials.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		49B2998A2409338200B7319C /* RLMApp.mm in Sources */ = {isa = PBXBuildFile; fileRef = 49B299862409338200B7319C /* RLMApp.mm */; };
+		49B2998B2409338200B7319C /* RLMApp.h in Headers */ = {isa = PBXBuildFile; fileRef = 49B299872409338200B7319C /* RLMApp.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		49B299902409351600B7319C /* app.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 49B2998C2409351500B7319C /* app.cpp */; };
+		49B299912409351600B7319C /* app.hpp in Headers */ = {isa = PBXBuildFile; fileRef = 49B2998D2409351500B7319C /* app.hpp */; settings = {ATTRIBUTES = (Public, ); }; };
+		49B299922409351600B7319C /* app_credentials.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 49B2998E2409351600B7319C /* app_credentials.cpp */; };
+		49B299932409351600B7319C /* app_credentials.hpp in Headers */ = {isa = PBXBuildFile; fileRef = 49B2998F2409351600B7319C /* app_credentials.hpp */; settings = {ATTRIBUTES = (Public, ); }; };
+		49B299962409351B00B7319C /* generic_network_transport.hpp in Headers */ = {isa = PBXBuildFile; fileRef = 49B299942409351B00B7319C /* generic_network_transport.hpp */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
These headers are in the wrong section, which indicates that they're probably incorrectly part of the Realm target when they shouldn't be.